content/posts/0009-php-rearchitecure.md

```diff
@@ -0,0 +1,107 @@
+re-architected and is ready to make containerizing PHP applications simple and
+transparent. With the rearchitecture, the same base set of functionality is
+supported as in the original PHP buildpack, but the buildpacks have been
+modernized using the newest Paketo tooling and modularized to separate concerns
+into individual buildpacks.
+
+## What's new?
+
+#### Breaking it down
+The rearchitecture has made it vastly easier for maintainers and contributors
+to add new functionality to the language family in the future, by breaking down
+what used to be three larger buildpacks, into 10+ new buildpacks responsible
+for single parts of the build process. This may sound like a lot of new
+buildpacks, but small lighter-weight buildpacks are actually what we strive
+for in the project. This concept is well-described in the [Paketo Buildpacks
+Philosophy Part 1 blog
+post](https://blog.paketo.io/posts/buildpack-philosophy-part-1/).
+
+The GIF below gives an overview of how the buildpacks have been split up based
+off of their main job. The new buildpacks are split into three order groups,
+one for the Nginx use case, one for the HTTPD use case, and one for the PHP
+built-in server use case. The order groups in the buildpack can be found in the
+[PHP buildpack `buildpack.toml`
+file](https://github.com/paketo-buildpacks/php/blob/main/buildpack.toml).
+
+![Rewrite GIF](/images/posts/0009/rewrite.gif)
+
+
+#### API Upgrades
+The buildpacks have also been re-written to leverage the newest Paketo
+tooling, such as the [packit
+library](https://github.com/paketo-buildpacks/packit). Using the packit library
+allows the buildpacks to run on the latest versions of the upstream Cloud
+Native Buildpacks specifiction to make sure our buildpacks support the full
+range of new features in the buildpacks ecosystem. The new PHP buildpacks are
+built on [Buildpack API
+0.7](https://github.com/buildpacks/spec/blob/buildpack/v0.7/buildpack.md), as
+opposed to the API 0.4 that the old buildpacks used.
+
+#### SBOM support
+Because of the API upgrade, the buildpacks now include [Software Bill of
+Materials](https://paketo.io/docs/concepts/sbom/) (SBOM) support. The SBOM
+provides metadata about the dependencies associated with your application
+image, and can be used for vulnerability scanning. Check out the [Paketo
+documentation](https://paketo.io/docs/howto/sbom/) for how to access the SBOM.
+
+#### Maintaining the same use cases
+If you're not already acquainted with the previous version of the buildpack PHP
+buildpack, the following use cases are supported:
+
+*  Build PHP apps that are served with HTTPD
+*  Build PHP apps that are served with Nginx
+*  Build PHP apps that are served the PHP Built-in Server
+*  Leverage the PHP FastCGI Process Manager (FPM) to run alongside servers
+*  Build apps that use Composer as an application level package manager
+*  Configure and run PHP session handlers for Redis and Memcached instances
```

Comment:
Maybe link to any corresponding samples? The reader could then run the sample and see each use case in action.

Reply:
good idea. Just added.